From 248874b8eb5e6cd216efd2a7d50689a3be0639c7 Mon Sep 17 00:00:00 2001
From: Fabian Diener <noreply>
Date: Thu, 22 Feb 2018 18:09:01 +0100
Subject: [PATCH] Build elastix as shared lib

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9bba09a7..4e6ffa18 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -32,7 +32,7 @@ include( CTest )
 #---------------------------------------------------------------------
 # Generate executable, static library or dynamic library.
 mark_as_advanced( ELASTIX_BUILD_EXECUTABLE )
-option( ELASTIX_BUILD_EXECUTABLE "Generate executable or library?" ON )
+option( ELASTIX_BUILD_EXECUTABLE "Generate executable or library?" OFF )
 
 if( NOT ELASTIX_BUILD_EXECUTABLE )
   add_definitions( -D_ELASTIX_BUILD_LIBRARY )
@@ -43,7 +43,7 @@ if( NOT ELASTIX_BUILD_EXECUTABLE )
   #set( CMAKE_CXX_VISIBILITY_PRESET hidden )
 
   mark_as_advanced( BUILD_SHARED_LIBS )
-  option( BUILD_SHARED_LIBS "Build shared libraries?" OFF )
+  option( BUILD_SHARED_LIBS "Build shared libraries?" ON )
   if( BUILD_SHARED_LIBS )
     add_definitions( -D_ELASTIX_USE_SHARED_LIBRARY )
 
-- 
2.16.2

